src/state/configure-store.ts

```diff
@@ -4,14 +4,29 @@ import { createLogicMiddleware } from "redux-logic";
 
 import { BASE_API_URL, DATA_BUCKET_URL } from "../constants";
 
-import { enableBatching, initialState, metadata, selection, State } from "./";
+import {
+    enableBatching,
+    initialState,
+    trajectory,
+    selection,
+    viewer,
+    simularium,
+    State,
+} from "./";
 
 const reducers = {
-    metadata: metadata.reducer,
+    trajectory: trajectory.reducer,
     selection: selection.reducer,
+    viewer: viewer.reducer,
+    simularium: simularium.reducer,
```

Comment:
are these last two actually new?  just wondering why they were absent before and added now

Reply:
Oh oops, yes this IS actually new! 😅 Previously there were only 2 branches (`metadata` and `selection`). Now we have 4. See https://github.com/allen-cell-animated/simularium-website/issues/166#issuecomment-887147239

Reply:
I see.  I guess my real thought is: in this file, it looks like these two lists really need to accumulate up the reducers and logics from every state branch.  Is that right?

Reply:
Yes. That is correct. This is where they all come together into one unified state tree